index.html

```diff
@@ -66,57 +66,23 @@
 		<!-- Banner -->
 			<section id="banner">
 				<header>
-					<h2>Sean Finney: <em> A professional software developer</em></h2>
+					<h2>Sean Finney: <em> A professional software engineer</em></h2>
 					<!-- <a href="https://docs.google.com/document/d/1A3xs0opHZLwXxvJ8L5qYv9V04TsZppfqGv-20xlEvb0/edit" class="button">My Resume</a> -->
 				</header>
 			</section>
 
-		<!-- Highlights -->
-			<!-- <section class="wrapper style1">
-				<div class="container">
-					<div class="row 200%">
-						<section class="4u 12u(2)">
-							<div class="box highlight">
-								<i class="icon major fa-paper-plane"></i>
-								<h3>Greedly</h3>
-								<p>An RSS feed reader that I made. I made it with a backbone frontend that use </p>
-							</div>
-						</section>
-						<section class="4u 12u(2)">
-							<div class="box highlight">
-								<i class="icon major fa-pencil"></i>
-								<h3>Also Important</h3>
-								<p>Duis neque nisi, dapibus sed mattis et quis, nibh. Sed et dapibus nisl amet mattis, sed a rutrum accumsan sed. Suspendisse eu.</p>
-							</div>
-						</section>
-						<section class="4u 12u(2)">
-							<div class="box highlight">
-								<i class="icon major fa-wrench"></i>
-								<h3>Probably Important</h3>
-								<p>Duis neque nisi, dapibus sed mattis et quis, nibh. Sed et dapibus nisl amet mattis, sed a rutrum accumsan sed. Suspendisse eu.</p>
-							</div>
-						</section>
-					</div>
-				</div>
-			</section> -->
-
 		<!-- Gigantic Heading -->
 			<section class="wrapper style2">
 				<div class="container">
 					<header class="major">
-						<h2>I live in San Francisco and I'm looking to make a start</h2>
-						<p>Ever since my first software developing experience with HTML in high school, I have been interested in programming. 
-							My professional experience started with building a program for a startup that interfaced heavily with a SQL database. After spending two years as a chemical engineer my interests turned back to coding. Now, I am moving back into the field of professional development by being a web developer.</p>
+						<h2>I am an experienced DevOps Engineer</h2>
+						<p>Ever since I ran into my first line of code in high school, I have always been fascinated by programming and how computers can be manipulated by scripts	

```

Comment:
First line of code => I love this

nd how computers can be manipulated by script => This could be wordsmithed, I wouldn't think you would be amiss to say something generic about how you can build things or whatever